convert/convert/converters/coco/coco_export/convert.py

```diff
@@ -347,7 +357,7 @@ def convert_to_coco(
             for element in page["objs"]:
                 if element["category"] not in category_names:
                     response = requests.post(
-                        settings.category_service_url,
+                        str(settings.category_service_url),
```

Comment:
It is `str` anyways. It defaults to `None` if not setup, then `'None'` is propagated to `requests.post`, which will cause "inappropriate" error for this

Reply:
Done